src/pages/EditMonitor.vue

```diff
@@ -621,6 +696,16 @@
                                 </div>
                             </div>
 
+                            <div v-if="monitor.type === 'websocket-upgrade' " class="my-3 form-check">
+                                <input id="ws-ignore-headers" v-model="monitor.wsIgnoreHeaders" class="form-check-input" type="checkbox">
+                                <label class="form-check-label" for="ws-ignore-headers">
+                                    {{ $t("Ignore Sec-WebSocket-Accept header") }}
+                                </label>
```

Comment:
lets be specific here in terms of variable naming to avoid confusion in the future.
This does not really ignore all headers.

This also needs to be replaced downstream

```suggestion
                                <input id="ignore-sec-websocket-accept-header" v-model="monitor.ignoreSecWebsocketAcceptHeader" class="form-check-input" type="checkbox">
                                <label class="form-check-label" for="ignore-sec-websocket-accept-header">
                                    {{ $t("Ignore Sec-WebSocket-Accept header") }}
                                </label>
```